src/key_defs.h

```diff
@@ -66,45 +66,45 @@ typedef union Key_ {
 
 
 
-#define KEY_FLAGS         B00000000
-#define CTRL_HELD         B00000001
-#define LALT_HELD         B00000010
-#define RALT_HELD         B00000100
-#define SHIFT_HELD        B00001000
-#define GUI_HELD          B00010000
-#define SYNTHETIC         B01000000
-#define RESERVED          B10000000
-
-#define LCTRL(k)  ((Key) { k.keyCode, k.flags | CTRL_HELD })
-#define LALT(k)   ((Key) { k.keyCode, k.flags | LALT_HELD })
-#define RALT(k)   ((Key) { k.keyCode, k.flags | RALT_HELD })
-#define LSHIFT(k) ((Key) { k.keyCode, k.flags | SHIFT_HELD })
-#define LGUI(k)   ((Key) { k.keyCode, k.flags | GUI_HELD })
+#define KEY_FLAGS         uint8_t(B00000000)
+#define CTRL_HELD         uint8_t(B00000001)
+#define LALT_HELD         uint8_t(B00000010)
+#define RALT_HELD         uint8_t(B00000100)
+#define SHIFT_HELD        uint8_t(B00001000)
+#define GUI_HELD          uint8_t(B00010000)
+#define SYNTHETIC         uint8_t(B01000000)
+#define RESERVED          uint8_t(B10000000)
+
+#define LCTRL(k)  ((Key) { k.keyCode, uint8_t(k.flags | CTRL_HELD) })
+#define LALT(k)   ((Key) { k.keyCode, uint8_t(k.flags | LALT_HELD) })
+#define RALT(k)   ((Key) { k.keyCode, uint8_t(k.flags | RALT_HELD) })
+#define LSHIFT(k) ((Key) { k.keyCode, uint8_t(k.flags | SHIFT_HELD) })
+#define LGUI(k)   ((Key) { k.keyCode, uint8_t(k.flags | GUI_HELD) })
 
 // we assert that synthetic keys can never have keys held, so we reuse the _HELD bits
-#define IS_SYSCTL                  B00000001
-#define IS_INTERNAL                B00000010
-#define SWITCH_TO_KEYMAP           B00000100
-#define IS_CONSUMER                B00001000
+#define IS_SYSCTL                  uint8_t(B00000001)
+#define IS_CONSUMER                uint8_t(B00000010)
```

Comment:
That is something, I hardly have an explanation for. Will be fixed asap.

Reply:
I am still not sure how this happend, but it seems strangely related to this:
https://github.com/keyboardio/Kaleidoscope/commit/f57aab5f718d3b47ea2252ec6c006c96cc62eb59